src/snake.h

```diff
@@ -23,14 +24,9 @@ public slots:
     void setGameState(GameState s);
     void updateBoundingRect(const QRectF& rect);
     void startNewGame();
-
-signals:
```

Comment:
This was broken because Snake was not a `Q_OBJECT`, my mistake. I will need this to work for the menu though.

Reply:
We can still merge this. I'll fix the rest on my branch.